cmd/semTable.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+    "fmt"
+    "github.com/anaskhan96/soup"
+)
+
+// semTable function - equivalent to your Python code
+func semTable(url string) {
+    fmt.Println("Fetching semesters...")
+    resp := fetchHTML(url)  // Fetch the HTML content
+
+    // Check if the response is empty
+    if resp == "" {
```

Comment:
A more solid comparison?

```suggestion
    if len(resp) == 0 {
```

Reply:
This code snippet seems to be updated in latest version.